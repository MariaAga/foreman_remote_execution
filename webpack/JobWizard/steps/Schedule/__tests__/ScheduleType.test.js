import React from 'react';
import * as patternfly from '@patternfly/react-core';
import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
import { ScheduleType } from '../ScheduleType';

jest.spyOn(patternfly, 'FormGroup');
patternfly.FormGroup.mockImplementation(props => <div props={props} />);
const fixtures = {
  renders: {},
};

describe('ScheduleType', () => {
  testComponentSnapshotsWithFixtures(ScheduleType, fixtures);
});
