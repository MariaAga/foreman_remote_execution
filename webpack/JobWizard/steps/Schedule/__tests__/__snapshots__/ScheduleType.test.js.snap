// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleType renders 1`] = `
<mockConstructor
  fieldId="schedule-type"
  label="Schedule type"
>
  <Radio
    className=""
    id="schedule-type-now"
    isChecked={true}
    isDisabled={false}
    isValid={true}
    label="Execute now"
    name="schedule-type"
    onChange={[Function]}
  />
  <Radio
    className=""
    id="schedule-type-future"
    isChecked={false}
    isDisabled={false}
    isValid={true}
    label="Schedule for future execution"
    name="schedule-type"
    onChange={[Function]}
  />
</mockConstructor>
`;
